languages/erlang/docs/implementing-a-concept-exercise.md

```diff
@@ -32,4 +32,15 @@ languages
                 └── rebar.config
 ```
 
+## Step 1: adding track-specific files
+
+These files are specific to the Erlang track:
+
+- `src/<NAME>.erl`: the stub implementation file, which is the starting point for students to work on the exercise.
+- `src/<NAME>.app.src`: the applicatioin definition which is pretty much the same across all exercises except for the application name.
```

Comment:
applicatioin -> application
I'm not sure what an application definition is--a kind of project file?--, but I assume that most potential contributors would know.

Reply:
On the "BEAM" VM the word "application" describes what one would call a "package" in many other languages, or "module" in go, "gem" in ruby.

I'm pretty sure this is worth a "concept" description later on.